servers/rendering/renderer_scene_cull.h

```diff
@@ -1154,9 +1154,14 @@ class RendererSceneCull : public RenderingMethod {
 	PASS1RC(float, environment_get_white, RID)
 
 	// Fog
-	PASS10(environment_set_fog, RID, bool, const Color &, float, float, float, float, float, float, float)
+	PASS11(environment_set_fog, RID, bool, RS::EnvironmentFogMode, const Color &, float, float, float, float, float, float, float)
+
+#ifndef DISABLE_DEPRECATED
+	PASS10(environment_set_fog_compat_84792, RID, bool, const Color &, float, float, float, float, float, float, float)
```

Comment:
Also unsure if this is correct, it should only be bound directly on the exposed class, the internals and how it works isn't relevant